core/src/main/java/dk/alexandra/fresco/framework/DRes.java

```diff
@@ -1,9 +1,9 @@
 package dk.alexandra.fresco.framework;
 
 /**
- * Is the root of all calculations in fresco.
+ * Is the root of all calculations in FRESCO.
  * This is intended to open up for some protocols to be implemented by producers.
- * Both NativeProtocol and ProtocolProducer can be a computation
+ * Both {@link NativeProtocol} and intermediate results can be a DRes.
  *
```

Comment:
This is quite central class. I think the documentation here really needs improvement. For example, you may want to mentioned that DRes is short for "deffered result" and that is what this class is exactly supposed to model. I think the documentation still has some reference to "computation" as well.

Reply:
True indeed - I actually did not know what the abbreviation was. Now I can do this in more detail.